src/account/AccountFactory.sol

```diff
@@ -82,7 +114,21 @@ contract AccountFactory is Ownable {
         return Create2.computeAddress(getSalt(owner, salt, entityId), _PROXY_BYTECODE_HASH);
     }
 
+    function getAddressFallbackSigner(address owner, uint256 salt) public view returns (address) {
```

Comment:
For symmetry, could we call this `getAddressSemiModularAccount` / `getAddressSemiModular`?

Reply:
Addressed here! [9d43bde](https://github.com/erc6900/reference-implementation/pull/123/commits/9d43bde5a7c52f9e9e2238b55edd9d65c3664c7b)